compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIViewComposeSceneLayer.uikit.kt

```diff
@@ -62,22 +62,26 @@ internal class UIViewComposeSceneLayer(
     private val backgroundView: UIView = object : UIView(
         frame = CGRectZero.readValue()
     ) {
-        init {
-            translatesAutoresizingMaskIntoConstraints = false
-            rootView.addSubview(this)
-            NSLayoutConstraint.activateConstraints(
-                getConstraintsToFillParent(this, rootView)
-            )
-        }
-
-        override fun pointInside(point: CValue<CGPoint>, withEvent: UIEvent?): Boolean {
-            //TODO pass invoke(true) on touch up event only when this touch event begins outside of layer bounds.
-            // Also it should be only one touch event (not multitouch with 2 and more touches).
-            // In other cases pass invoke(false)
-            onOutsidePointerEvent?.invoke(PointerEventType.Press)
-            onOutsidePointerEvent?.invoke(PointerEventType.Release)
-            return focusable
-        }
+        private var previousSuccessHitTestTimestamp: Double? = null
```

Comment:
You don't need to add any timestamps here

Reply:
Discussed it - we still need `previousSuccessHitTestTimestamp` to call `onOutsidePointerEvent?.invoke(PointerEventType.Press)` just once on one touch